scheduler/kubernetes/scheduler-convergence.md

```diff
@@ -3,6 +3,9 @@ layout: page
 title: "Run pods on same host as a volume"
 keywords: portworx, container, Kubernetes, storage, Docker, k8s, flexvol, pv, persistent disk, StatefulSets
 sidebar: home_sidebar
+redirect_from:
+  - /kubernetes-with-scheduler-convergence.html
+  - /scheduler/kubernetes/mysql.html
```

Comment:
https://docs.portworx.com/scheduler/kubernetes/failover.html

This is the closest page we have